### PR TITLE
Backport 1.1 sensu fixes

### PR DESCRIPTION
**Pull request by dlundquist**

### CODE REVIEW

File: roles/monitoring-common/defaults/main.yml [hunk 1]
```diff
@@ -11,4 +11,4 @@ monitoring:
     vhost: /sensu
     username: sensu
     password: sensu
-  cluster_prefix: "ip a | grep :ucarp > /dev/null &&"
\ No newline at end of file
+  cluster_prefix: "ip a | grep :ucarp > /dev/null"
```
Comment (j2sol) [inline roles/monitoring-common/defaults/main.yml]: There is still something wrong with this. The && is supposed to be there according to @paulczar

Comment (dlundquist) [inline roles/monitoring-common/defaults/main.yml]: Maybe we should just write a nice wrapper:
`if_active_controller /path/to/metrics-nova-oversub.sh`

**Comment by paulczar** [comment]
something went horribly wrong with my backport.   I'll redo it in the morning.

**Comment by paulczar** [comment]
The original backport has been redone here - #842

**Comment by dlundquist** [comment]
#842 has been merged, abandoning this one.

**Pull request closed without merging**